OracleManagementAgent/dockerfiles/latest/Dockerfile

```diff
@@ -25,13 +25,16 @@
 #		$ bash build.sh
 #
 
-FROM oraclelinux:7-slim
+FROM container-registry.oracle.com/os/oraclelinux:8-slim
 
 # Labels
 LABEL "provider"="Oracle"                                               \
       "issues"="https://github.com/oracle/docker-images/issues"         \
       "volume.data"="/opt/oracle"
 
+# Install yum package manager
+RUN microdnf -y install yum
```

Comment:
This is not acceptable. If you want to use `yum` then you must switch to the `oraclelinux:8` base image and not use the `slim` variant.

Reply:
Use `ghcr.io/oracle/oraclelinux:8-slim`